File: 1848-sum-of-unique-elements/sum-of-unique-elements.cpp
class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        unordered_map<int,int> mp;
        for(int i:nums){
            mp[i]++;
        }
        int sum=0;
        for(auto entry:mp){
            if(entry.second==1){
                sum+=entry.first;
            }
        }
        return sum;
    }
};

File: 1848-sum-of-unique-elements/sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        HashMap<Integer,Integer> mp=new HashMap<>();
        for(int i:nums){
            mp.put(i,mp.getOrDefault(i,0)+1);
        }
        int sum=0;
        for(Map.Entry<Integer,Integer> entry:mp.entrySet()){
            if(entry.getValue()==1){
                sum+=entry.getKey();
            }
        }
        return sum;
    }
}